Library/Formula/pgroonga.rb

```diff
@@ -0,0 +1,39 @@
+class Pgroonga < Formula
+  desc "PostgreSQL plugin to use Groonga as index"
+  homepage "https://pgroonga.github.io/"
+  url "http://packages.groonga.org/source/pgroonga/pgroonga-1.0.0.tar.gz"
+  sha256 "854d66bdc79e7cfceb29ae8706556a3ad1c8b32ac4535939d296966e3b19fd3e"
+
+  depends_on "pkg-config" => :build
+  depends_on "groonga"
+  depends_on "postgresql"
+
+  def install
+    # To find HOMEBREW_PREFIX/bin/pg_config
+    ENV.prepend_path "PATH", HOMEBREW_PREFIX/"bin"
```

Comment:
IMO, I like dividing formulae instead of all in one formulae.
Because optional resourced extensions are not handled during `upgrade/unlink/link/switch`.

In fact, I've confused that `PGroonga` does not to be handled when upgrading postgresql.